src/ZODB/scripts/fsrefs.py

```diff
@@ -118,30 +119,38 @@ def main(path=None):
     # This does not include oids in undone.
     noload = {}
 
-    for oid in fs._index.keys():
+    #print("# building references graph ...")
+    graph = {} # oid -> refs
+
+    posoidv = list((pos, oid) for (oid, pos) in fs._index.items()) # [] of (pos, oid)
+    posoidv.sort() # access objects in order of pos↑  (optimize disk IO)
```

Comment:
Can we avoid the use of unnecessary non-ascii characters?
```suggestion
    posoidv.sort() # access objects in order of ascending file position  (optimize disk IO)
```

Reply:
ok